test/AspNetCoreModule.Test/FunctionalTestHelper.cs

```diff
@@ -35,7 +35,7 @@ public bool IsMet
         {
             get
             {
-                if (TestFlags.Enabled(TestFlags.SkipTest))
+                if (_attributeValue == TestFlags.SkipTest)
```

Comment:
@jkotalik I wanted to allow we can set multiple values for the _attributeValue with ";" delimiter such as "RunAsAdministrator;SkipTest". In that case, we can't use "==" here.

Reply:
I disagree with allowing that for attributes. What else would we allow to be set? Anyways the check here https://github.com/aspnet/AspNetCoreModule/blob/dev/test/AspNetCoreModule.Test/FunctionalTestHelper.cs#L44 is also incorrect then. I am keeping this consistent with what is already there.

Reply:
For example, the websocket functional tests will be only applicable only when websocket iis module installed because the websocket module is not available on downlevel OS such as Win7.
In that case, the test case should be set with "RunAsAdministrator;WebSocketModuleAvailable" supposing the test requires the "RunAsAdministrator" environment.

Reply:
No, we have OS version rules for our WebSocket tests.

Reply:
No we can't use OS version, because, even in the latest OS, the IIS websocket module is not enabled by default.

Reply:
Do you have negative tests for how it fails when the module is missing? If not then make installing the module a system prerequisite so you don't have to worry about it in the test. Our tests have several system prereqs.

Reply:
Let's handle this in another PR if we want to support this kind of behavior.